src/ActivityLogger.php

```diff
@@ -177,7 +171,7 @@ public function log(string $description)
         return $activity;
     }
 
-    public function withoutLogs(callable $callback)
+    public function withoutLogs(callable $callback): mixed
```

Comment:
`callable` is too broad and I see no reason to pass in `[$this, 'method']` or any other `callable`.

```suggestion
    public function withoutLogs(Closure $callback): mixed
```